kotlin-dsl-layoutcontainer/src/main/java/com/hannesdorfmann/adapterdelegates4/dsl/LayoutContainerListAdapterDelegateDsl.kt

```diff
@@ -115,6 +117,7 @@ internal class DslLayoutContainerListAdapterDelegate<I : T, T>(
  *
  * @since 4.1.0
  */
+@ContainerOptions(cache = CacheImplementation.HASH_MAP)
```

Comment:
What do you think about using `SPARSE_ARRAY` instead of `HASH_MAP`?

Reply:
In my opinion in this case it does not matter. `HASH_MAP` is selected because this is the default implementation. Change to `SPARSE_ARRAY`?